### ISSUE TITLE
Website Loading Issue

### ISSUE THREAD

**Issue by num47** [pH7Software/pH7-Social-Dating-CMS#518]
Hello,
I have installed pH7CMS on my server. But it is not loading correctly. Tried it like 100 times. But, getting the same problem. My website link is [https://ox69.net/hizuppo/](https://ox69.net/hizuppo/) You can check it yourself how it looks like. I have contacted with the hosting provider and they said they have checked it and their server meets all the requirement for pH7CMS. 

Can anyone please give me a solution. 

Thank a lot.

**Comment by TGHepburn** [comment]
num47;

I'm having exact same issue.  Just did an install on new server/host.  I used Softaculous (because I was lazy).  I'm going to re-install using manual setup of DB and dl from Sourceforge.  

Seems that one of the CSS files is not loading. Admin area is fine.

**Comment by TGHepburn** [comment]
Something to do with Cache settings under "tools"....?

**Comment by polynamaude** [comment]
@num47 You can first start by disabling the gzip compression in the options.
https://github.com/pH7Software/pH7-Social-Dating-CMS/wiki/%5BDRAFT%5D-Problem-Getting-Assets-(CSS-JS)-cURL

https://github.com/pH7Software/pH7-Social-Dating-CMS/issues/106

That should give you some hint for investigation.

If not then

https://github.com/pH7Software/pH7-Social-Dating-CMS/wiki/How-to-ask-for-help

**Comment by polynamaude** [comment]
@num47 Also, please make sure you aren't running mod_security or modsecurity-crs on your server as it may cause unpredictable results.

**Comment by num47** [comment]
@TGHepburn I actually have tried to install it both manually and using softaculous. None worked. You are a little better position than i am 'cos my admin panel is also loading the same way. And, what cache settings actually you were talking about? Is that a solution for this problem? Would you please explain a little if changing some settings would solve the problem? I have actually installed it on my own localhost server and it works perfectly fine there. 

@polynamaude Thanks a lot for the suggestion. I will check it and see if things get fixed. Will update you.

**Comment by num47** [comment]
@polynamaude added SetEnv no-gzip 1 to .htaccess to turn off the gzip. I am not sure if that's not a proper code to disable the gzip or it still not working after disabling gzip.

**Comment by polynamaude** [comment]
@num47 Did you take the time to read the three links I've sent ?
Look over if modsecurity-crs is enabled on the server ?

At last, there's a nice series of hint that tells you how to ask for help (if you do want to get helped). And now you're damn far from there.

https://github.com/pH7Software/pH7-Social-Dating-CMS/wiki/%5BDRAFT%5D-Problem-Getting-Assets-(CSS-JS)-cURL

I don't see nowhere something related to using **_SetEnv no-gzip 1_** !!!!
If you don't want our answer then don't waste our time.

**Comment by num47** [comment]
It worked finally! only disabling the gzip probably is enough. Wrote the following code on the .htaccess file and it laded nicely. The code is - 
`php_flag zlib.output_compression off
`  

Though i first tried to use the following to disable the gzip and mod security. But those didn't work. And, apparently for me i do not need to use these following codes either. 

SetEnv no-gzip 1

<IfModule mod_security.c>
  SecRuleEngine Off
  SecFilterInheritance Off
  SecFilterEngine Off
  SecFilterScanPOST Off
  SecRuleRemoveById 300015 3000016 3000017
</IfModule>


@polynamaude thanks a lot. 
@TGHepburn you can try this and see if this fixes your problem.

**Comment by TGHepburn** [comment]
Last night I did a fresh install and things got worse somehow.  After re-install, behavior matched @num47 exactly: back-end and front-end had malformed pages.
@num47 Your suggestions did not work regarding modifications to the .htaccess file.  I was referring to the administration features which are shown in the attached GIF showing what happens when Gzip is disabled.  This has some effect on front end but does not help. 

I have another shared host site which has installation of V15 working sweetly ( funio.com ).  The shared host with issues is hostupon.ca (one of the recommended hosting companies by pH7 ) .  Thus I am able to compare settings.  

- ModSecurity is OFF on both sites (though Cpanel )
- Working has PHP 7.0; Site with issues has PHP 7.1 ( I will change this and report back time permitting)
- PHP packages are the same (I'll add screenshot soon )
- No changes have been made to the .htaccess file on working site.
- both installed through Softaculous
- both in root directory of domain/sub domain
- working url : http://dating.whodidthis.ca
- issues url: http://www.flirtbywire.net

![Capture-18](https://user-images.githubusercontent.com/23179549/56442509-47cd9e00-62ae-11e9-989f-54d0ddd5ee3e.gif)
![firefox_LUhubPwY8b](https://user-images.githubusercontent.com/23179549/56442870-739d5380-62af-11e9-9318-df6e0f3bcb2c.png)

I am going to hunt down any log files that may exist on this CMS as I have not read any mention of them yet.

**Comment by polynamaude** [comment]
@TGHepburn Either I'm missing something or didn't I gave a link called "how to get help" ?

Maybe with some logs we can try to see what's going on.

Just go into the administrative interface and disable all the cache type. This should help out.

There's also some other closed issue with the same cache problem. Look at them.

**Comment by num47** [comment]
@TGHepburn You have to delete the cache after you use `php_flag zlib.output_compression off`in your .htaccess

You will find the cache in `/_protected/data/cache/` 

And, the links @polynamaude shared actually will give you more insight about this problem.

**Comment by TGHepburn** [comment]
@num47 Thanks for the tip.  I took it one step further and deleted everything.  The long weekend put a halt to my efforts but it was a good thing.  Turns out that the hosting company had not fully configured the server space.  The solution was to post ticket with hosting company.  They were quick to respond.  Now there _are_ Apache errors showing up and the site behaves normally.  I'm sure it's due to my lack of knowledge.  I've read the links provided by @polynamaude  and will re-read again.  Thank-you for the help.
